charts/ratify/values.yaml

```diff
@@ -76,6 +76,12 @@ oras:
       endpoint: ""
       partition: "" # defaults to aws
       region: ""
+  cache:
+    enabled: true
+    ttl: 10  # in seconds
+    capacity: 100 # Megabytes
+    keyNumber: 10000 # maximum key number
```

Comment:
Not a blocker for this PR, but please document the significance of these values in a follow up.